config/locales/en/providers.yml

```diff
@@ -1580,7 +1581,7 @@ en:
       multi_proceeding: Proceeding %{position} of %{total}
       client_involvement_type:
         show:
-          question: What is your client’s role in this proceeding?
+          question: What is your client's role in this proceeding?
```

Comment:
There are a lot of changes which look like they are just to do with the apostrophe - is this because these were previously backticks instead of apostrophes? Finding it a bit hard to tell just by looking!

Reply:
Yes. I was trying to fix some of the tests that were failing because of the `’` vs `'` and I just ended up changing all of them rather than only the specific ones that I needed to make the specific failing tests pass.